.travis.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+# Copyright (c) 2021, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Is there any specific reason to change the year?

Reply:
Misunderstood the usage of this year in license header. Reverted it.